tests/_support/Helper/Acceptance.php

```diff
@@ -0,0 +1,63 @@
+<?php
+namespace Psalm\MockeryPlugin\Tests\Helper;
+use Codeception\Exception\Skip;
+use Codeception\Exception\TestRuntimeException;
+use Composer\Semver\Comparator;
+use Composer\Semver\VersionParser;
+use Muglug\PackageVersions\Versions;
+// here you can define custom actions
+// all public methods declared in helper class will be available in $I
+class Acceptance extends \Codeception\Module
+{
+    /** @var array<string,string */
+    const VERSION_OPERATORS = [
+        'newer than' => '>',
+        'older than' => '<',
+    ];
+
+    /**
+     * @Given /I have Psalm (newer than|older than) "([0-9.]+)" \(because of "([^"]+)"\)/
+     *
+     * @return void
+     */
+    public function havePsalmOfACertainVersionRangeBecauseOf(string $operator, string $version, string $reason)
```

Comment:
These are copies from the PHPUnit version of this from the psalm phpunit plugin: https://github.com/psalm/phpunit-psalm-plugin/blob/master/tests/_support/Helper/Acceptance.php#L26 just with the repo change for the respective library I'm checking the version of.